Library/Formula/libagar.rb

```diff
@@ -0,0 +1,58 @@
+class Libagar < Formula
+  desc "Cross-platform GUI toolkit"
+  homepage "http://libagar.org/"
+  url "http://stable.hypertriton.com/agar/agar-1.4.1.tar.gz"
+  sha256 "b0e62b754f134c3c0dd070a4fa62fa552654356eebab3d55e32d5d9b151a275e"
+  head "http://dev.csoft.net/agar/trunk", :using => :svn
+
+  depends_on "sdl"
+  depends_on "freetype"
+  depends_on "jpeg"
+  depends_on "libpng"
+
+  def install
+    # Parallel builds failed to install config binaries
+    # https://bugs.csoft.net/show_bug.cgi?id=223
+    ENV.deparallelize
+
+    system "./configure", "--prefix=#{prefix}"
+    system "make"
+    system "make", "install", "MANDIR=#{man}" # --mandir for configure didn't work
+  end
+
+  test do
+    (testpath/"test.c").write <<-EOS.undent
+      #include <agar/core.h>
+      #include <agar/gui.h>
+
+      int main() {
+        AG_Window *win;
+        if (AG_InitCore("test", AG_VERBOSE) == -1 || AG_InitGraphics(NULL) == -1) {
+          return 1;
+        } else {
+          return 0;
+        }
+      }
+    EOS
+    flags = %W[
+      -I#{include}/agar
+      -I#{Formula["sdl"].include}/SDL
```

Comment:
The `opt` folders are symlinks back to the formula's keg, but by following the opt prefixes instead of the Cellar paths it means we test against whatever is linked. Generally `opt_something` is the preferred way to point at another formula's installation, aye.